lib/MessagesClient/index.js

```diff
@@ -27,14 +27,32 @@ MessagesClient.sendMessageWithAttachment = (
   fileName,
   fileType,
   fileContent,
-  format
+  format,
+  sessionToken
 ) => {
   if (format === MessagesClient.PRESENTATIONML_FORMAT) {
     message = '<div data-format="PresentationML" data-version="2.0">' + message + '</div>'
-    return send(conversationId, message, data, fileName, fileType, fileContent)
+    return send(conversationId, message, data, [{fileName, fileType, fileContent}], sessionToken)
   } else if (format === MessagesClient.MESSAGEML_FORMAT) {
     message = '<messageML>' + message + '</messageML>'
-    return send(conversationId, message, data, fileName, fileType, fileContent)
+    return send(conversationId, message, data, [{fileName, fileType, fileContent}], sessionToken)
+  }
+};
+
+MessagesClient.sendMessageWithMultipleAttachments = (
+  conversationId,
+  message,
+  data,
+  files,
```

Comment:
does this have any public documentation? 
we should be clair on the format of the files array